.github/workflows/build-images.yml

```diff
@@ -0,0 +1,77 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+  release:
+  workflow_dispatch:
+
+jobs:
+  build-container-image:
+    name: Image Build ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: Debian
+            file: Docker/Dockerfile
+            flavor: |
+              latest=auto
+              suffix=
+              onlatest=false
+          - name: Alpine
+            file: Docker/Dockerfile-Alpine
+            flavor: |
+              latest=auto
+              suffix=-alpine
+              onlatest=true
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
```

Comment:
The checkout can be removed from the action as long as `docker/build-push-action` has no context option defined. It will pull the repo by itself and has no need for a checkout.